services/node_update.proto

```diff
@@ -148,4 +149,14 @@ message NodeUpdateTransactionBody {
      * If set, the new value SHALL replace the existing hash value.
      */
     google.protobuf.BytesValue grpc_certificate_hash = 7;
+
+    /**
+     * A new administrative key for the node address book entry.
+     * <p>
+     * If this field is set, that key MUST sign this transaction.<br/>
+     * If this field was previously set, that key MUST also sign
```

Comment:
This will always be set previously right since its required on `node_create`

Reply:
Updated.